fuse/ipns/ipns_test.go

```diff
@@ -131,9 +189,45 @@ func TestFilePersistence(t *testing.T) {
 	}
 }
 
+func TestDeeperDirs(t *testing.T) {
```

Comment:
this isn't a very deep test... it's just 2-3 dirs. with a name like `DeeperDirs` i'd expect a stress test.